numpy/distutils/checks/cpu_avx512f.c

```diff
@@ -2,6 +2,11 @@
 
 int main(void)
```

Comment:
we already testing this bug through `_mm512_castsi512_si128()` which is missing too on gcc4.9.

Reply:
Didn't realize that. I reverted changes in `cpu_avx512f.c`.